frontend_new/src/components/ThemeSetting.tsx

```diff
@@ -0,0 +1,16 @@
+
+function getDefaultTheme() {
+    //get the current date
+    var date = new Date();
```

Comment:
**[Required, Note]** Should avoid using `var` as much as possible. This is because you can redeclare vars like so:
```javascript
var somevar = 10;
var somevar = "test";
```

This is confusing and makes it hard to tell what the type is or where the original was defined. It's also dangerous when it comes to local and global scoping.

Instead of using `var`, use `let` and if you need a constant then use `const`